src/box/tuple_format.c

```diff
@@ -166,12 +166,19 @@ tuple_field_delete(struct tuple_field *field)
 
 /** Return path to a tuple field. Used for error reporting. */
 static const char *
-tuple_field_path(const struct tuple_field *field)
+tuple_field_path(const struct tuple_field *field, const struct tuple_format *format)
```

Comment:
I would prefer this change in different commit.. But I'm not sure. Please try to do it if it's not hard to do. I hope there are not many test that are affected by this change.
I mean that actually the task consists of two parts - print actual type and print name instead of index. Those tasks seem to be independent. It's better to solve independent task in independent commits.

Reply:
Divided into 2 commits.

Reply:
Divided into 2 commits.